packages/beacon-node/src/sync/backfill/backfill.ts

```diff
@@ -503,10 +503,10 @@ export class BackfillSync extends (EventEmitter as {new (): BackfillSyncEmitter}
    * the initialization point is the same block tree as the DB once backfill
    */
   private async checkIfCheckpointSyncedAndValidate(): Promise<void> {
-    if (this.syncAnchor.lastBackSyncedBlock == null) {
+    if (this.syncAnchor.lastBackSyncedBlock === null) {
```

Comment:
type `this.syncAnchor.lastBackSyncedBlock` is `BackfillBlock | null`. Compiler ensure this won't be null so safe to do strict checking against `null`